temporal-sdk/src/main/java/io/temporal/internal/common/WorkflowExecutionUtils.java

```diff
@@ -98,11 +98,10 @@
 
   private static final Logger log = LoggerFactory.getLogger(WorkflowExecutionUtils.class);
 
-  private static final RpcRetryOptions retryParameters =
+  private static final RpcRetryOptions GET_INSTANCE_CLOSE_EVENT_RETRY_OPTIONS =
       RpcRetryOptions.newBuilder()
-          .setBackoffCoefficient(2)
-          .setInitialInterval(Duration.ofMillis(500))
-          .setMaximumInterval(Duration.ofSeconds(30))
+          .setBackoffCoefficient(1)
+          .setInitialInterval(Duration.ofMillis(1))
```

Comment:
Unrelated to this PR, since you've just copied existing values, but is it a good idea to hammer server without backoff?

Reply:
Thank you for rising it!
It's actually not and this is already on my radar and I probably will send one more PR shortly after I familiarize myself better with code implementations of GrpcRetryer and BackoffThrottler.
This issue was discovered specifically because clients very aggressively hammering the server.